CONTRIBUTING.md

```diff
@@ -160,27 +168,43 @@ Uninstalling keras-nightly-2.5.0.dev2021032500:
 
 ## Run tests
 
+We use [Bazel](https://bazel.build/) to build and run the tests.
+
+### Run a test file
+
+For example, to run the tests in `keras/engine/base_layer_test.py`,
+we can run the following command at the root directory of the repo.
+
+```shell
+bazel test keras/engine:base_layer_test
+```
+
+`keras/engine` is the relative path to the directory containing the test file.
```

Comment:
This is the BUILD package path, which is bazel specific. You can have a target like a:b_test, but the actual test python file lives in a/b/c_test.py